notNeededPackages.json

```diff
@@ -3853,6 +3853,10 @@
             "libraryName": "mem",
             "asOfVersion": "4.2.0"
         },
+        "mem-fs-editor": {
+            "libraryName": "mem-fs-editor",
+            "asOfVersion": "10.0.0"
```

Comment:
Version 10.0.0 appears broken because the types in package.json points to `dist/index.d.ts`, but there wasn't a `dist` folder. So I think this should be 10.0.1, where they fixed this.

Reply:
Fixed.